### ISSUE TITLE
Android Studio crashing on set scale in xml file

### ISSUE THREAD

**Issue by jadermoraes** [addisonElliott/SegmentedButton#17]
Hi!
My IDE keep crashing when I set the scaleX on a SegmentedButton.

I'm using Ubuntu.
Thank you, the lib is working great besides this error.

**Comment by addisonElliott** [comment]
Hello.

It's odd that it ends up crashing Android Studio itself. To make this easier, do you mind posting an example setup that reproduces the issue? In addition, the stack trace of the issue, if available, would be good.

**Comment by jadermoraes** [comment]
[MyApplication2.zip](https://github.com/addisonElliott/SegmentedButton/files/3569615/MyApplication2.zip)
I'm sorry, but couldn't get the log, the IDE crashes when i set the scalex in the xml file.

**Comment by addisonElliott** [comment]
No problem, this should be enough.

FYI, I expect this may take a few weeks to resolve. Anything you can do to temporarily keep working will be good. My main machine has a failing SSD that I'm focusing on resolving.

**Comment by jadermoraes** [comment]
No problem, i worked around that, and the lib is working amazingly, thanks for the support!

**Comment by addisonElliott** [comment]
@jadermoraes I finally got some time to debug and play around with your issue.

I tested your exact application on my Windows 10 machine and an Ubuntu 18.04 VM. I was able to change the `scaleX` parameter on the button group and buttons themselves with no crash in Android Studio.

This is likely an Android Studio bug and not an issue with the library. In theory, regardless of what code I have in my library, it shouldn't cause the IDE to crash.

I'm going to close this now. I recommend (if you are committed to getting it fixed), reporting it to Google to look into.
